fuseki/sample-ubd.trig

```diff
@@ -164,7 +191,10 @@ graph <cube-project/ubd/cube-data> {
     sh:path      cube:observedBy .
 
   _:pollutant
-    sh:in        ( <https://environment.ld.admin.ch/foen/ubd/28/pollutant/so2> ) ;
+    sh:in        (
+                   "so2" # <https://environment.ld.admin.ch/foen/ubd/28/pollutant/so2>
+                   <https://environment.ld.admin.ch/foen/ubd/28/pollutant/co>
+                 ) ;
```

Comment:
Finally, the Cube Shape would be a mix of mapped and unmapped value until all mappings are created.